### ISSUE TITLE
[Suggestions] Disable view model motion blur (and also maybe for the ship and cruiser?)

### ISSUE THREAD

**Issue by zndrmn** [RiskiVR/Lethal_MotionBlur#1]
I think it would be great if we could have the option to disable motion blur for the view model (hands, visor, etc.)
Also would it be possible to disable motion blur for the ship when taking off? I also believe there could be weird motion blur issues when sitting in the back or ontop of the cruiser.

~~sorry for the really bad bitrate, github only allows you to upload 10 mb videos so yeah~~

Also I set the motion blur strength to `0.15` so it's not as obvious as it is on `1.2`

<details>
<summary>Videos showing motion blur on the view model when jumping/running & when the ship is moving</summary>

https://github.com/user-attachments/assets/42591011-ceab-4332-91f6-fecdaea0b4ea

https://github.com/user-attachments/assets/623cea14-c540-4a0f-bb02-5c08fb01f921

</details>

**Comment by RiskiVR** [comment]
I did a small attempt to try removing the blur on the visor by setting it's motion vectors to none, however that didn't make a difference. I would love to see what issues you're having with the provided videos, however with the intensity at 0.15 I can't really tell what's going on at all.

**Comment by zndrmn** [comment]
Yeah I should probably get better videos with a higher bitrate hehe. 
In the first video the entire ship gets blurred and looks a little weird, looking at the top of the visor it's a little more noticeable. 
The second video was mainly just showcasing the motion blur on the view model. (the top of the visor getting blurred just looks a little weird to me, which could just be me)

**Comment by RiskiVR** [comment]
Ohh yeah I know what you mean. The blurring when the ship lands is pretty bad yeah, that's mainly due to the entire object moving while your view is static, in which case your view camera doesn't know that, it just sees the ship as an object moving, therefore it's calculating those motion vectors and just blurring it's motion. 

As for the visor blur, I cannot figure out how to exclude it from being blurred since the way motion vectors are used and calculated, it blurs your entire vision in order to blur that area of where the motion is happening. The reason why the HUD isn't affected at all is because the HUD are UI elements, which are on a completely different layer, and that layer is not affected by the post process volume.

**Comment by Paradox75831004** [comment]
@RiskiVR Or maybe just add a check - when you're on the ship during landing and takeoff - Motion Blur works, if not, it doesn't work. Is it possible to do this?

**Comment by RiskiVR** [comment]
@Paradox75831004 If there is a function that is called on ship landing and takeoff, I could add a patch that disables mb on ship landing and takeoff.
